src/main/java/domain/Lotto.java

```diff
@@ -20,16 +20,17 @@ public Lotto(String input) {
     }
 
     public Lotto(List<Integer> lottoNumbers) {
-        validateLottoNumberCount(lottoNumbers);
-        Collections.sort(lottoNumbers);
+        Set<Integer> numbers = new HashSet<>(lottoNumbers);
+        validateLottoNumberCount(numbers);
 
         this.lottoNumbers = lottoNumbers.stream()
+                .sorted()
                 .map(LottoNumber::new)
-                .collect(Collectors.toList());
+                .collect(Collectors.toSet());
     }
 
-    private static void validateLottoNumberCount(List<Integer> lottoNumbers) {
-        if (lottoNumbers.stream().distinct().count() < LOTTO_NUMBER_COUNT_LIMIT) {
+    private static void validateLottoNumberCount(Set<Integer> lottoNumbers) {
+        if (lottoNumbers.stream().count() < LOTTO_NUMBER_COUNT_LIMIT) {
```

Comment:
그냥 사이즈를 바로 꺼낼 수 있지 않을까요 ㅎㅎ
```suggestion
                if (lottoNumbers.size() < LOTTO_NUMBER_COUNT_LIMIT) {
```